assets/src/scss/_dependencies.scss

```diff
@@ -29,7 +28,7 @@
  * be accessed via mixins
  *
  */
-
+@import 'settings/variables';
```

Comment:
Was this just to make sure the asset path resolves correctly?

Reply:
yep.

Reply:
Should it go under a new heading? Doesn't seem like it should live under a "Fozzie Extensions" heading?

Reply:
Yeh - it should do.  I think the comment above should be associated with the `card` include below, and then this dependency should have it's our section above this comment but below the `f-icons` include.